src/components/SQForm/useFormButton.tsx

```diff
@@ -3,7 +3,21 @@ import {useFormikContext} from 'formik';
 import {useDebouncedCallback} from 'use-debounce';
 import {hasUpdated} from '../../utils';
 
-export function useFormButton(isDisabled, shouldRequireFieldUpdates, onClick) {
+type UseFormButtonReturnType = {
+  isButtonDisabled: boolean;
+  hasFormBeenUpdated: boolean;
+  values: unknown;
+  initialValues: unknown;
+  isValid: boolean;
+  handleClick: (...args: unknown[]) => void;
+  [key: string]: unknown;
+};
+
+export function useFormButton(
+  isDisabled: boolean,
+  shouldRequireFieldUpdates: boolean,
+  onClick?: (event: React.ChangeEvent<HTMLInputElement>) => void
```

Comment:
I think the MouseEvent type you previously had was correct. The onClick type that Mui expects for the Button component is `React.MouseEventHandler<HTMLButtonElement>`, so the onClick type in SQFormButton and SQFormIconButton need to be changed to match that.

Reply:
ahh cool! I was thinking the same thing but since the SQFormIconButton got merged I assumed that was the correct one, ill go ahead and adjust that.

Reply:
Yeah, that was an oversight 🥴